content/docs/studio/user-guide/billing-and-payment.md

```diff
@@ -37,57 +39,50 @@ team. All invoice amounts are calculated for one billing period at a time. A
 
 If you add members to available seats or remove members without reducing the
-number of seats in your team, there will be no impact in the cost to you.
+number of seats in your team, there will be no price impact.
 
 If you increase the number of seats, your account will be charged a prorated
 amount based on the time remaining in the billing period at the time each seat
 was added. For instance, if your billing period runs from the 15th of last month
 to the 14th of this month, then, if you add 2 seats on the 10th of this month,
-you will be charged for only 5 days for the 2 new seats. In the following
-billing period, the 2 additional seats will be taken into account for the entire
-billing period.
+you will be charged for only 5 days of the 2 new seats. In the following billing
+period, the 2 additional seats will be charged for the entire billing period.
 
-If you reduce the number of seats, you will accordingly receive an account
-credit that you can apply on your next payment.
+If you reduce the number of seats, you will receive an account credit for the
+prorated difference. You can apply it for a future payment.
 
 ## Downgrading a plan
 
-At any point during your subscription, you can reduce the number of seats in
-your plan.
+You can reduce the number of seats in your plan at any point, as explained in
+the previous section.
 
-To cancel the Team plan and move to the Free plan, reduce the number of seats in
-the team to 1 or 2 and click on Downgrade. When you reduce the number of seats
-in the team, a credit balance will be computed based on the number of days
-remaining in the billing period. This credit balance can be used in future
-payments.
+To cancel the Team plan and move back to the Free plan, reduce the number of
+seats in the team to 1 or 2 and click on `Downgrade`. A prorated credit will be
+added to your account for the remaining days in the billing period. You can
+apply for a future payment.
 
 When you downgrade from the Team plan to the Free plan, you will receive a
-refund based on the number of days remaining in the billing period. For any
-inquiries about refunds, please [contact us](mailto:support@iterative.ai).
+prorated refund for the days remaining in the billing period. For inquiries
+about refunds, please [contact us].
 
-To upgrade to or downgrade from an Enterprise plan, please
-[contact us](mailto:support@iterative.ai).
+To upgrade to or downgrade from an Enterprise plan, please [contact us].
 
-## Billing period, auto-renewal of team plan, and grace period for non-payment
+## Billing period, auto-renewal of Team plan, and grace period for non-payment
 
 The billing period for each team plan subscription starts on the date of the
-subscription, and is one month long. For example, if your subscription date is
+subscription and is one month long. For example, if your subscription date is
 the 15th of May, your next billing period will start on the 15th of June.
```

Comment:
What happens if you sign up on a 31st? Or on Feb 28th? @tapadipti 🙏🏼

Reply:
Not entirely sure. @ssachkovskaya do you know what the billing end date is in this case?

Reply:
This question is outstanding but shouldn't block this PR (can be clarified in a later one).